javalib/src/test/java/com/strumenta/kolasu/javalib/CompilationUnit.java

```diff
@@ -22,6 +22,12 @@ public List<B> getBs() {
         public List<PropertyDescription> getProperties() {
             return Arrays.asList(new PropertyDescription("bs", true, Multiplicity.MANY, getBs(), PropertyType.CONTAINMENT, false));
         }
+
+        @Override
+        @NotNull
+        public List<PropertyDescription> getOriginalProperties() {
```

Comment:
Is it necessary to always override this method in Java? Or only when overriding _getProperties_?

Reply:
It is currently always required, let me change that